command/server.go

```diff
@@ -518,6 +518,7 @@ func (c *ServerCommand) Run(args []string) int {
 			sealType := vaultseal.Shamir
 			if !configSeal.Disabled && os.Getenv("VAULT_SEAL_TYPE") != "" {
 				sealType = os.Getenv("VAULT_SEAL_TYPE")
+				configSeal.Type = sealType
```

Comment:
Do you know why we are setting sealType to shamir above? seems like we always are overriding it with this if/else

Reply:
Probably just to make sure the type is correct?

Reply:
Yea, I think it's just to make sure that it's a valid default value that we can start off with.